.github/workflows/releases.yml

```
@@ -132,23 +132,18 @@ jobs:
           unzip -o *.zip && rm -v *.zip
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-      - name: Install go-msi
-        run: choco install -y "go-msi"
       - name: Prepare PATH
-        shell: bash
-        run: |
-          echo "$WIX\\bin" >> $GITHUB_PATH
-          echo "C:\\Program Files\\go-msi" >> $GITHUB_PATH
+        id: setupmsbuild
+        uses: microsoft/setup-msbuild@v1.0.3
       - name: Build MSI
         id: buildmsi
         shell: bash
         env:
           ZIP_FILE: ${{ steps.download_exe.outputs.zip }}
+          MSBUILD_PATH: ${{ steps.setupmsbuild.outputs.msbuildPath }}
         run: |
-          mkdir -p build
-          msi="$(basename "$ZIP_FILE" ".zip").msi"
-          printf "::set-output name=msi::%s\n" "$msi"
-          go-msi make --msi "$PWD/$msi" --out "$PWD/build" --version "${GITHUB_REF#refs/tags/}"
+          name="$(basename "$ZIP_FILE" ".zip")"
+          "${MSBUILD_PATH}\MSBuild.exe" .\build\windows\gh.wixproj -p:SourceDir="$PWD" -p:OutputPath="$PWD" -p:OutputName="$name" -p:ProductVersion="${GITHUB_REF#refs/tags/v}"
```

How did all this work before? Let's try this now:
```suggestion
          "${MSBUILD_PATH}\MSBuild.exe" ./build/windows/gh.wixproj -p:SourceDir="$PWD" -p:OutputPath="$PWD" -p:OutputName="$name" -p:ProductVersion="${GITHUB_REF#refs/tags/v}"
```

---

github.com/AlecAivazis/survey/v2 library for user input ... so I'm gonna guess this bug is really something wrong in that library

Edit: I can repeatably reproduce this now, with this sequence:

1. I have a systemd user unit that sets up a screen session, and re-launches it if it exits
2. When I attach this locally via

---

github.com/AlecAivazis/survey/v2 library for user input ... so I'm gonna guess this bug is really something wrong in that library

Edit: I can repeatably reproduce this now, with this sequence:

1. I have a systemd user unit that sets up a screen session, and re-launches it if it exits
2. When I attach this locally via